app/views/admin/chapters/organisers/index.html.haml

```diff
@@ -1,28 +1,33 @@
-%section
+.container-fluid
   .row
-    .medium-12.columns
-      %h2
+    .col
+      %h1
         Organisers
         %small.subheader= @chapter.name
-    .medium-6.columns
-      = simple_form_for :organiser, url: admin_chapter_organisers_path(@chapter), html: { class: 'simple_form' } do |f|
-        %fieldset
-          %legend Add new organiser
-          = f.select :organiser, @chapter_members.uniq.map { |member| ["#{member.full_name}", member.id] }, { include_blank: true }, { class: 'chosen-select', required: true, data: { placeholder: 'Select a member to make organiser' } }
-          = f.submit 'Add organiser', class: 'button expand'
+  .row
+    .col-12.col-md-6.mt-3
+      = simple_form_for :organiser, url: admin_chapter_organisers_path(@chapter), html: { class: 'border border-primary p-3' } do |f|
+        %label.form-label.mb-2 Add new organiser
+        = f.select :organiser, @chapter_members.uniq.map { |member| ["#{member.full_name}", member.id] }, { include_blank: true }, { class: 'form-select form-select-lg', required: true, data: { placeholder: 'Select a member to make organiser' } }
+        = f.submit 'Add organiser', class: 'btn btn-primary btn-block mt-3'
+
+  .row.mt-4.organisers
```

Comment:
I believe the test is failing because it is looking for the newly added content inside this scope, `.organisers`:
```rb
      within '.organisers' do
        expect(page).to have_content(chapter_subscriber.full_name)
      end
```

If you wrap all these `.row`s on a `.organisers` container, this should be fixed (and a bit more organised, I recon).